### ISSUE TITLE
Code Table Request - frame rate

### ISSUE THREAD

**Issue by wellerjes** [ArctosDB/arctos#7783]
### Goal

create an attribute to record physical media

### Context

creating new attributes that can be applicable for cultural collections and archives

### Table

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type

~~https://arctos.database.museum/info/ctDocumentation.cfm?table=cttissue_volume_units~~

### Proposed Value

frame rate

~~fps~~

~~frames per second~~

 - add fps as a search term

### Proposed Definition

Frame rate (commonly expressed in frames per second or FPS) is typically the frequency at which consecutive images are captured or displayed. This definition applies to film and video cameras, computer animation, and motion capture systems.

### Collection type

AV, ART, ES, ARCH

number+units

n/a

#### Attribute units

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcount_units

### Example Data

For CHAS:AV, most of the films are between 16 to 24 frames per second, but this changes with sound film and more modern technology

### Helpful Actions

- [x] Add the issue to the [Code Table Management Project](https://github.com/ArctosDB/arctos/projects/13#card-31628184).

- [ ] Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary.

@ArctosDB/arctos-code-table-administrators 

## Approval
_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality). **No person should act in multiple roles**; the submitter cannot also serve as a Code Table Administrator, for example._

- [x] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [x] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [x] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [x] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection
_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation
_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

- [ ] Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel.

- [ ] Add or revise the code table term/definition as described above. Ensure the URL of this Issue is included in the definition. URLs should be included *_as text_*, separated by spaced pipes. Do not include HTML in definitions.

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by Jegelewicz** [comment]
> FPS (frames per second)

Would need to be added to a units table. I suggest https://arctos.database.museum/info/ctDocumentation.cfm?table=cttissue_volume_units

**Comment by wellerjes** [comment]
Updated the request to add new unit of measurement.

**Comment by dustymc** [comment]
Units table as proposed is not acceptable, see https://github.com/ArctosDB/arctos/issues/7791

Possibly we need a 'ratio' or similar (hasn't that been discussed somewhere?), in the meantime https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcount_units seems to be a useful catchall.

(And I prefer 'frames per second' over 'FPS (frames per second)' when we find a home; those parens will find a way to be annoying.)

**Comment by Jegelewicz** [comment]
Changed request to include

_if numerical values should be accompanied by units, provide a link to the appropriate units table._

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcount_units

~~FPS (frames per second)~~

per second - values indicate a rate per second

OR

rate - values indicate a rate that should be described in the associated term (for example: frames per second or miles per hour)

**Comment by dustymc** [comment]
I think we need "frames" even if we eventually end up with lots of ' xxx per second' things. (When given the opportunity, I'll always assume light years...)

I'll open an issue for that.

**Comment by Jegelewicz** [comment]
> Maybe change this to "frame rate"???

I had the same thought....which is why I went where I went.

**Comment by Jegelewicz** [comment]
I propose that we change this to frame rate for consistency with Audubon Core

https://ac.tdwg.org/termlist/#ac_frameRate

with units that are 

`per second` - the related attribute is measured in units per second

we could call it inverse second - but that's just too esoteric - https://en.wikipedia.org/wiki/Inverse_second

>The inverse second or reciprocal second (s−1), also called per second, is a [unit](https://en.wikipedia.org/wiki/Unit_of_measurement) defined as the [multiplicative inverse](https://en.wikipedia.org/wiki/Multiplicative_inverse) of the [second](https://en.wikipedia.org/wiki/Second) (a unit of [time](https://en.wikipedia.org/wiki/Time)). It is applicable for [physical quantities](https://en.wikipedia.org/wiki/Physical_quantities) of [dimension](https://en.wikipedia.org/wiki/Dimension_(physics)) reciprocal time, such as [frequency](https://en.wikipedia.org/wiki/Frequency) and [strain rate](https://en.wikipedia.org/wiki/Strain_rate).

**Comment by mkoo** [comment]
I think we're overthinking this. ```frames per second``` is a standard measure of frame rate and even if it's not originally listed as such it is often converted to fps to facilitate comparison and usually so it can be converted to the standard 24 fps for viewing. In other words, fps will likely be available. So like with ```uncertainty in meters```, we just convert if we have extents or uncertainty listed in another unit. I think it would be good to have a consistent rate to facilitate comparisons and tracking in the database.

**Comment by wellerjes** [comment]
I would like to proceed with frames per second. Thanks!

**Comment by dustymc** [comment]
I'm completely lost at this point, and need the full proposal spelled out somewhere. (I think we're at "frames per second per second" at the moment - a measure of acceleration?!?!)

**Comment by Jegelewicz** [comment]
> I think we're at "frames per second per second" at the moment

I think that is true and if the people who will be using it think this works, then let's accelerate!

My alternative is

`frame rate` with units of `per second`

**Comment by wellerjes** [comment]
> frame rate with units of per second

Works for me!

**Comment by dustymc** [comment]
>frame rate with units of per second

"frame rate=60 per second"

I don't love it (but I don't think it's completely wrong/fatal either).

I think I still prefer where I landed in https://github.com/ArctosDB/arctos/issues/7783#issuecomment-2155323797

* attribute_type=``frame rate``
* attribute_units=``frames per second``

which reads as 

"frame rate=60 frames per second"

**Comment by mkoo** [comment]
CT meeting= Go with Dusty's suggestion above

**Comment by Jegelewicz** [comment]
@dustymc please confirm the revised request. @wellerjes @droberts49 will the revised request work for you guys?

**Comment by wellerjes** [comment]
Yes, please proceed.

**Comment by wellerjes** [comment]
What code table is this part of? I'm trying to add it as an attribute for CHAS:AV and I'm not finding it, [just the units that it's tied to](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcount_units).

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
This did not get added to the attribute type code table.

**Comment by AJLinn** [comment]
I think this got merged into the [frames per second](https://github.com/ArctosDB/arctos/issues/7887) request... am I remembering right @mkoo ?
